test/setup_transfer.cpp

```diff
@@ -1140,6 +1140,53 @@ void stop_web_server()
 	web_server_pid = 0;
 }
 
+namespace {
+pid_type websocket_server_pid = 0;
+}
+
+int start_websocket_server(bool ssl, int min_interval)
+{
+	int port = 2000 + static_cast<int>(lt::random(6000));
+	error_code ec;
+	io_context ios;
+
+	// make sure the port we pick is free
+	do {
+		++port;
+		tcp::socket s(ios);
+		s.open(tcp::v4(), ec);
+		if (ec) break;
+		s.bind(tcp::endpoint(make_address("127.0.0.1")
+			, std::uint16_t(port)), ec);
+	} while (ec);
+
+	std::string python_exe = "python3"; // python 3 is required
```

Comment:
I think it would be better to use `get_python()` here. The purpose of that function isn't to pick python2 or python3, it's to handle a special case I had to implement on appveyor to pick the right python interpreter when running with mingw. Now, mingw was too much work to get to work in C++17, but I still think it makes sense to maintain this abstraction for future CI updates

Reply:
Indeed, I wrote this when `get_python()` targeted python 2.